credits.txt

```diff
@@ -16,8 +16,11 @@ Developers
   Amazinite
   Benjamin Hauch
   MCOfficer
+  Peter van der Meer
   Pointedstick
+  Quyykk
   W1zrad
+  Zitchas
 
 Major Programming
   AdamKauffman
```

Comment:
If you're renaming the mention from the contributor list, update this one too.

Reply:
This person has deleted their Github account.